decimal64.go

```diff
@@ -22,7 +22,7 @@ const (
 
 func signalNaN64() Decimal64 {
 	// TODO: What's the right behavior?
-	panic("sNaN64")
+	return SNaN64
```

Comment:
Still haven't found a suitable way to signal the calling function without panicking